trackAnalysisStatistics_automizer_config.ini

```diff
@@ -0,0 +1,55 @@
+#works for both, Dmin_autocalculator and batchAnalysisStatistics. Make sure you set min_detectable_D in DIFFUSION_TYPE_PARAM to the value you get from the Dmin_autocalculator
```

Comment:
"# Execution of trackAnalysis and trackStatistics for multiple coverslips at once."